package.json

```diff
@@ -53,6 +54,7 @@
     "eslint-plugin-promise": "^6.0.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
+    "lodash": "^4.17.21",
```

Comment:
Is this a large dependency? I want to keep the SDKs lean and mean. I saw you are just using the `times` function to replace a for-loop. I don't feel that justifies the increased asset size.

Reply:
i added it as a `devDependency`, so shouldn't affect the actual package

Reply:
oh cool I didn't notice that - sounds good!